packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -144,6 +144,59 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Text::H3>With different types of links as content</Shw::Text::H3>
+
+  <Shw::Grid @columns={{2}} as |SG|>
+    <SG.Item>
+      <Hds::Alert @type="inline" @color="success" as |A|>
+        <A.Title>HTML link compared to the <code>Hds::Link</code> in the description</A.Title>
+        <A.Description>
+          <p>Description with an <a href="#">HTML link</a>.</p>
+          <p>
+            Compared with a
+            <Hds::Link::Inline @href="#">Primary Hds::Link::Inline</Hds::Link::Inline>, and a
+            <Hds::Link::Standalone @icon="hashicorp" @text="Primary Hds::Link::Standalone with an icon" @href="#" />
+          </p>
+          <p>
+            Also compared with a
+            <Hds::Link::Inline @href="#" @color="secondary">Secondary Hds::Link::Inline</Hds::Link::Inline>, and a
+            <Hds::Link::Standalone
```

Comment:
[blcoker] I don't think we should use a `Link::Standalone` in the `Description` block of an alert, that's not what is intended for (so it's not a real use case, and doesn't make sense to "test" it in the showcase).

Also the other occurrences below.